Automattic/jetpack
# Donations: Swap display of currency control

When choosing the currency for donations display it as name - symbol
rather than symbol - name. This makes it consistent with the settings
for Payments and Pay with Paypal, as well as looking slightly nicer.

Fixes #17025





#### Jetpack product discussion


 p58i-9k2-p2#comment-47441.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:






1. Ensure you're on a paid wp.com plan (or a jetpack complete plan if testing on jetpack)
2. Create a new post
3. Add a Donations block to the post
4. Look at the currency setting for the block, it should display the currency three letter code followed by the currency symbol
5. Click on the currency setting, likewise the whole list should display name then symbol

Before | After
-------|------
 <img width="790" alt="currency-order-before" src="https://user-images.githubusercontent.com/93301/148804631-4538fa79-1c22-42ae-b87e-0441d57c09c4.png"> | <img width="785" alt="currency-order-after" src="https://user-images.githubusercontent.com/93301/148804661-1ca5cc09-0c04-42c6-a878-164101dfb9aa.png">
